src/Cli/dotnet/commands/dotnet-test/Program.cs

```diff
@@ -56,10 +57,10 @@ public static int Run(ParseResult parseResult)
                 return ForwardToVSTestConsole(parseResult, args, settings, testSessionCorrelationId);
             }
 
-            return ForwardToMsbuild(parseResult, settings, testSessionCorrelationId);
+            return ForwardToMsbuild(parseResult, args, settings, testSessionCorrelationId);
```

Comment:
Forgot to cleanup this part:

```suggestion
            return ForwardToMsbuild(parseResult, settings, testSessionCorrelationId);
```